gradle-plugin/src/main/kotlin/xyz/block/artifactswap/ArtifactSwapSettingsPlugin.kt

```diff
@@ -89,18 +89,9 @@ class ArtifactSwapSettingsPlugin : Plugin<Settings> {
    */
   private fun Settings.setupArtifactSwapInfrastructure(bomVersion: String) {
     logger.info("Artifact Swap BOM version: {}", bomVersion)
-
-    gradle.settingsEvaluated {
-      // This has to run after settings are evaluated. It ensures that the project descriptor
```

Comment:
this comment was only true when we used to do the rewriting of the buildscripts instead of the dynamic metaprogramming trick